app/src/main/java/com/example/myapplication/SecondFragment2.java

```diff
@@ -405,44 +414,24 @@ public void onDestroyView() {
      * @param mode false for light mode, true for dark mode
      */
     public void lightDark (ViewGroup v, boolean mode){
```

Comment:
You're using the exact copy of lightDark here in multiple files, so when that happens it's good to simplify your code by putting it in an outside helper class and calling it from these files so that there's less redundancy